storage/tianmu/core/engine.cpp

```diff
@@ -484,14 +484,23 @@ void Engine::EncodeRecord(const std::string &table_path, int table_id, Field **f
           v = common::TIANMU_BIGINT_MAX;
         else if (v < common::TIANMU_BIGINT_MIN)
           v = common::TIANMU_BIGINT_MIN;
-        *(int64_t *)ptr = v;
+        *reinterpret_cast<int64_t *>(ptr) = v;
+        ptr += sizeof(int64_t);
+      } break;
+      case MYSQL_TYPE_BIT: {
+        int64_t v = f->val_int();
+        // DEBUG_ASSERT(v < 0, "bit type data should never less than 0.");
```

Comment:
why we DO NOT use the logic of `BIGINT`, rather than adding these lines.

Reply:
Emmm... you are right, I had deleted this useless comment.